### PR TITLE
[PHP] Document default extensions

### PR DESCRIPTION
**Pull request by J0WI**
https://github.com/friendica/docker/pull/78#issuecomment-529210776

### CODE REVIEW

File: php/content.md [hunk 1]
```diff
@@ -66,6 +65,45 @@ Remember, you must install dependencies for your extensions manually. If an exte
 
 See ["Dockerizing Compiled Software"](https://tianon.xyz/post/2017/12/26/dockerize-compiled-software.html) for a description of the technique Tianon uses for determining the necessary build-time dependencies for any bit of software (which applies directly to compiling PHP extensions).
 
+### Default extensions
+
+Some extensions are compiled by default. As of PHP 7.3 these are:
```
Comment (tianon) [inline php/content.md]: Is there a command we could suggest users run instead to get this list for themselves?

Comment (J0WI) [reply php/content.md]: I got the list with `php -m`

Comment (yosifkit) [reply php/content.md]: I'd rather document the command instead of the list; this list will too easily be out of date.

Comment (J0WI) [reply php/content.md]: Done.

**Pull request merged**